enhancements/windows-containers/disconnected-support.md

```diff
@@ -0,0 +1,439 @@
+3. A cluster administrator transitions a connected cluster *with existing mirror registry settings* to a disconnected cluster
+4. A cluster administrator transitions a connected cluster *without existing mirror registry settings* to a disconnected cluster
+
+
+In cases 1, 2 & 3, Windows nodes can be joined to the cluster after altering registry settings,
+which would result in WMCO applying mirror settings during initial node configuration. 
+
+In case 2, for any existing Windows nodes, WMCO will react by updating each instance's state to consume the new registry config.
+
+In case 3, existing Windows nodes will not be re-configured. Existing registry settings will stay applied.
+
+In case 4, WMCO will take no action as there are no settings to configure. Windows nodes will not be able to pull images
+until mirror registry settings are configured by the administrator.
+
+### Risks and Mitigations
+
+If node cleanup fails, instances may retain their containerd configuration. This could be a concern for BYOH instances
+particularly. An event should be generated to alert cluster admins manual cleanup is required to remove WMCO-created files.
+
+Another concern is if a user's mirror configuration has overlapping settings/clashes. As on the Linux side, it is upon
+the user to ensure there is no clash between mirrors across their IDMS and ITMS resources.
+
+<!--
+Containerd may not support all the fields available in IDMS/ITMS resources.
+Unsupported fields will be ignored and will not impact containerd registry config/default behavior.
+-->
+
+### Drawbacks
+
+The only drawbacks are the increased complexity of WMCO and the potential complexity of debugging customer cases that
+involve a custom mirror registry setup, since it would be harder to set up an accurate replication environment.
+This can be mitigated by proactively getting the development team, QE, and support folks familiar with the expected
+behavior of Windows nodes/workloads in disconnected clusters.
+
+#### Support Procedures
+
+In general the support procedures for WMCO will remain the same.
+Customers that need support with disconnected environments must upgrade to the minimum versions: OCP 4.15 and WMCO 10.15.0.
+
+### API Extensions
+
+N/A, as no CRDs, admission and conversion webhooks, aggregated API servers, or finalizers will be added or modified.
+Only the WMCO will be extended which is an optional operator with its own lifecycle and SLO/SLAs, a
+[tier 3 OpenShift API](https://docs.openshift.com/container-platform/4.12/rest_api/understanding-api-support-tiers.html#api-tiers_understanding-api-tiers).
+
+### Operational Aspects of API Extensions
+
+N/A
+
+#### Failure Modes
+
+N/A
+
+### Release Plan
+
+The feature associated with this enhancement is targeted to land in the official Red Hat operator version of WMCO 10.15.0
+within OpenShift 4.15 timeframe. The normal WMCO release process will be followed as the functionality described in this
```

Comment:
Shouldn't this be 10.16/OpenShift 4.16? This changes the upgrade and version strategy below.